hr_holidays_public/models/hr_holidays_public.py

```diff
@@ -108,15 +101,14 @@ def get_holidays_list(
         return holidays_lines
 
     @api.model
-    def is_public_holiday(self, selected_date, employee_id=None, partner_id=None):
+    def is_public_holiday(self, selected_date, partner_id=None):
         """
         Returns True if selected_date is a public holiday for the employee
         :param selected_date: datetime object
-        :param employee_id: ID of the employee
         :param partner_id: ID of the partner
         :return: bool
         """
-        partner = self._get_partner_deprecated_employee(partner_id, employee_id)
+        partner = self._get_partner(partner_id)
```

Comment:
IMO there is no need for the `_get_partner()` method, you can do it directly there `self.env["res.partner"].browse(partner_id)`